app/views/course/material/folders/show.html.slim

```diff
@@ -1,4 +1,4 @@
-= page_header(@folder.name)
+= page_header(display_name(@folder))
```

Comment:
OK, Will probably need a helper like `display_files(folder)` and also a view partial for it. I'll do it in future PRs. 

So do you want me to rename this function  ?